src/main/java/uk/nhs/hee/tis/trainee/forms/mapper/LtftMapper.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright 2025 Crown Copyright (Health Education England)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+ * associated documentation files (the "Software"), to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify, merge, publish, distribute,
+ * sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT
+ * NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package uk.nhs.hee.tis.trainee.forms.mapper;
+
+import static org.mapstruct.MappingConstants.ComponentModel.SPRING;
+
+import java.util.List;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import uk.nhs.hee.tis.trainee.forms.dto.LtftSummaryDto;
+import uk.nhs.hee.tis.trainee.forms.model.LtftForm;
+
+/**
+ * A mapper to convert between LTFT related entities and DTOs.
+ */
+@Mapper(componentModel = SPRING)
+public interface LtftMapper {
+
+  /**
+   * Convert a {@link LtftForm} entity to a {@link LtftSummaryDto} DTO.
+   *
+   * @param entity The entity to convert to a DTO.
+   * @return The equivalent summary DTO.
+   */
+  @Mapping(target = "programmeMembershipId", source = "programmeMembership.id")
+  LtftSummaryDto toSummaryDto(LtftForm entity);
+
+  /**
+   * Convert a list of {@link LtftForm} to {@link LtftSummaryDto} DTOs.
+   *
+   * @param entities The entities to convert to DTOs.
+   * @return The equivalent summary DTOs.
+   */
+  @Mapping(target = "programmeMembershipId", source = "programmeMembership.id")
```

Comment:
This mapping is redundant since it will call `toSummaryDto` multiple times anyway.